### DISCUSSION TITLE
Support for Kibana 8.x or latest kibana version?

### DISCUSSION THREAD

**Discussion by manoj30** [wazuh/wazuh-kibana-app#5221] [Q&A] [answered]
Plugin currently supports till 7.10. Kibana current version is 8.6.
What are approx. dates for support for Kibana 8.x or current kibana version?

**Comment by Tostti** [comment, marked as answer]
Hello @manoj30, thank you for using Wazuh! 

Currently, the plugin supports up to 7.17.6. You can see the [compatibility matrix](https://documentation.wazuh.com/current/upgrade-guide/compatibility-matrix/index.html#elastic-and-the-wazuh-kibana-plugin) for reference. In the first launch of version 4.4, it will support up to 7.17.8. 

We are currently working to allow Wazuh to be integrated with Kibana 8, but we don't have an ETA yet. 

Kind regards,

**Reply by manoj30** [reply]
OK. Thanks for info.

**Reply by lukas1505** [reply]
@Tostti are you still working to support Kibana Version 8 or is this no longer planned?

**Reply by AlexRuiz7** [reply]
@lukas1505 check our [integrations guide](https://documentation.wazuh.com/current/integrations-guide/index.html).
